libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -551,9 +551,9 @@ protected void onPostExecute(OAuth2.TokenEndpointResponse tr) {
                  */
                 if (!passcodeManager.hasStoredPasscode(mgr.getAppContext())) {
```

Comment:
the reason the boolean was added is because this line, because technically, you do have a stored passcode, so it should not enter this 'if', we anyways have already set the lock screen